integration-tests/verification-flow/bbs-plus-revocation.test.ts

```diff
@@ -195,6 +273,54 @@ describe('BBS+ revocation', () => {
     expect(certsResponse.verified).toBe(true);
   });
 
+  it('should verify a revokable bbs+ credential with an updated witness', async () => {
+    const wallet: IWallet = await getWallet();
+
+    getCredentialProvider().addCredential(credentialWithUpdatedWitness);
+
+    const proofRequest = await createProofRequest(
+      ProofTemplateIds.ANY_CREDENTIAL,
+    );
+
+    const result: any = await getCredentialProvider().isValid(credential);
+
+    // TODO: Uncomment this when gets done https://dock-team.atlassian.net/browse/DCKM-483
```

Comment:
is there a comment on that ticket to remind us to come back here?

Reply:
I just added a comment to https://dock-team.atlassian.net/browse/DCKM-483